src/dist/dist.rs

```diff
@@ -346,7 +366,13 @@ impl TargetTriple {
                 (b"Linux", b"arm") => Some("arm-unknown-linux-gnueabi"),
                 (b"Linux", b"armv7l") => Some("armv7-unknown-linux-gnueabihf"),
                 (b"Linux", b"armv8l") => Some("armv7-unknown-linux-gnueabihf"),
-                (b"Linux", b"aarch64") => Some(TRIPLE_AARCH64_UNKNOWN_LINUX),
+                (b"Linux", b"aarch64") => {
+                    if is_32bit_userspace() {
+                        Some("armv7-unknown-linux-gnueabihf")
+                    } else {
+                        Some(TRIPLE_AARCH64_UNKNOWN_LINUX)
+                    }
+                }
```

Comment:
Style nit: I'd probably write it like this, what do you think?

```suggestion
                (b"Linux", b"aarch64") => Some(match is_32bit_userspace() {
                    true => "armv7-unknown-linux-gnueabihf",
                    false => TRIPLE_AARCH64_UNKNOWN_LINUX,
                }),
```

Reply:
I think using `match` on bools is weird :-)

Happy to change if you prefer though.

Reply:
@djc Extracting `Some` is indeed better IMHO, but matching a boolean also looks weird to me... The intent is very clear either way.

Reply:
Did the `Some` part, let me me know if you want the `match` part as well.

Reply:
I know it's maybe not the majority opinion of the Rust ecosystem (IMO mostly for being unfamiliar), but I think it's objectively better -- one line less, and with both arms laid out right next to each vertically. Anyway, it's fine.

Reply:
@alex Thanks a lot for bearing with us! We have just gained the permission today after all 😂